docs/3.0.0-beta.x/community/contributing-guide.md

```diff
@@ -0,0 +1,75 @@
+# Open Development & Community Driven
+
+Strapi is an open-source project administered by [the Strapi team](https://strapi.io/company). We appreciate your interest and efforts in contributing to Strapi.
+
+There are many ways that you can contribute to Strapi. Contributions to Strapi and the greater Strapi Community can take the form of answering questions in one of the [many Strapi channels](/3.0.0-beta.x/community/contributing-guide.html#get-in-touch), writing articles, creating videos and otherwise publishing self-generated content.
+
+The two additional ways to contribute to Strapi include contributing **Documentation** and **Code**:
```

Comment:
Contributions can take several forms:

- helping the community on the Strapi channels
- adding code
- ...


This way you can simplify the paragraph above and remove this sentence.

Reply:
I simplified this sentence.